src/routes/base.routes.js

```diff
@@ -65,22 +68,22 @@ export class BaseRoutes {
    * @return {object}
    */
   view() {
-    this._validateControllerHandler('view')
+    this[ Symbol.for( 'validateControllerHandler' ) ]( 'view' );
 
     return {
-      method: 'GET',
-      path: this.endpoint + '/{id}',
-      handler: this.controller.view.bind(this.controller),
-      config: {
-        description: 'Get an entry by ID',
-        tags: [ 'public' ],
-        validate: {
-          params: {
-            id: this.joi.number().integer().required().description('ID, primary key')
-          }
-        }
-      }
-    }
+      method  : 'GET',
+      path    : `${this.endpoint}/{id}`,
+      handler : this.controller.view.bind( this.controller ),
+      config  : {
+        description : 'Get an entry by ID',
+        tags        : ['public'],
+        validate    : {
+          params : {
+            id : this.joi.number().integer().required().description( 'ID, primary key' ),
+          },
+        },
+      },
```

Comment:
It supposed to make the diffin easier, in nodejs is fine, in the frontend not because IE8.
Atom automatically added trailing commas based on eslint and Airbnb's style guide.